.github/workflows/validate.yml

```
@@ -1,18 +1,17 @@
-name: Validate cnames_active.js
+name: Validate cnames and PR Template
```

```suggestion
name: Validate
```
Nit: I think we can stick with a shorter name here